modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/services/HealthCheckConsensus.scala

```diff
@@ -0,0 +1,41 @@
+package org.tessellation.sdk.domain.healthcheck.services
+
+import cats.Monad
+import cats.effect.Ref
+import cats.syntax.bifunctor._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import org.tessellation.sdk.domain.healthcheck.types.ConsensusRounds
+
+abstract class HealthCheckConsensus[F[_]]()(implicit F: Monad[F]) extends HealthCheck[F] {
```

Comment:
I know that it's sometimes useful to have direct access to the type class instance, however in the rest of the code we pass those instances through type constraints, so for the consistency of the codebase I would prefer using `HealthCheckConsensus[F[_]: Monad]`

Reply:
Okay, I'll do as you want. IMO both syntaxes are okay and can be used interchangeably